test/publishToNpm.test.js

```diff
@@ -90,15 +100,18 @@ tap.test(
       'pack',
       '--dry-run',
     ])
+    t.pass('npm pack called')
```

Comment:
I added a lot of these as without at least one test, Tap's output doesn't say which test in a file failed